### ISSUE TITLE
Pk and folding motifs

### ISSUE THREAD

**Issue by skoc** [sinc-lab/sincFold#2]
Hi, thank you for you sharing your great work. Have a question regarding Supplementary Materials table 2. Couldn't find where you annotate pase pairs with listed pk and folding motif types for TSO and do the eval. Is it possible to share your evaluation scripts for that experiment?
Thanks in advance

**Comment by lbugnon** [comment]
Hi, in the case of PK we just measured the avg F1 of the sequences that contained any PK, although you could find the specific PK positions and measure the F1 there

**Comment by skoc** [comment]
I see thanks for the response. It is avg F1 of sequences which contains PK, not only PK detected base pairs. 

Then my question is if you get the avg F1 of whole sequence, then contributions of PKs will be minimal due to being minority class across RNA sequence?

**Comment by lbugnon** [comment]
yes, PK are a small number of  interactions, however it dependes on how you define PKs (if they are point interactions or large areas of knoted parts of the structure). This is why we just reported avg F1

**Comment by lbugnon** [comment]
Closing old discussion, reopen if needed
